static/sass/styles.scss

```diff
@@ -853,3 +853,13 @@ html {
 img.has-shadow {
   box-shadow: 0 0 1rem #e5e5e5;
```

Comment:
can this use one of the [existing ](https://github.com/canonical/vanilla-framework/blob/main/scss/_settings_placeholders.scss) box-shadows in vnailla?  If not, can you please make a wg proposal with this one, and mention why the existing ones aren't a good fit? Would help to keep things from becoming too fragmented and inconsistent between codebases.

Reply:
There is a noticeable difference between this one and the vanilla one, so it might be worth suggesting upstream. Would be good to get input on who made this change, I think @sowasred2012 was working on this last?